db/migrate/20160412142212_create_locations.rb

```diff
@@ -9,6 +9,6 @@ def change
     end
 
     add_foreign_key :flood_risk_engine_locations,
-                    :flood_risk_engine_addresses
+                    :flood_risk_engine_addresses, column: :address_id
   end
```

Comment:
@autotelik I don't think column is required if rails can derive it because - I think rails convention will step in cos the  column is :address_id ?